### DISCUSSION TITLE
[Enhancement]: Add the possibility to manage Volume and persistent storage of applications with Coolify API

### DISCUSSION THREAD

**Discussion by APKiosk-Developers** [coollabsio/coolify#4100] [Improvement Requests]
### Request Type

Improvement

### Description

Is it possible to add the possibility to manage volume and persistent storage with the Coolify API

**Comment by Yiin** [comment]
Would be nice, I'm trying to setup automated private setup and deployment of my projects using API and this is quite a blocker.

**Comment by e508ea3d** [comment]
Pretty big blocker to maintain automated app creation with volumes without API. And frustrating that web UI has own API to do this manually
